resources/com.chatterino.chatterino2.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2019 Artem Polishchuk <ego.cordatus@gmail.com> -->
```

Comment:
Is this compatible with the MIT license ([used by Chatterino 2](https://github.com/Chatterino/chatterino2/blob/master/LICENSE))?

Reply:
Yep, this appdata under `CC0-1.0` license which have no restrictions at all. Just a credit to author. I followed this guidelines
https://docs.fedoraproject.org/en-US/packaging-guidelines/AppData/#_appdata_xml_file_creation